samples/snippets/src/main/java/com/example/logging/ListLogs.java

```diff
@@ -28,35 +28,64 @@
 /** List logs programmatically using the Cloud Logging API. */
 public class ListLogs {
 
-  /** Expects an existing Cloud Logging log name as an argument. */
   public static void main(String... args) throws Exception {
+
+    printLogNames();
+
+    /** Expects an existing Cloud Logging log name as an argument. */
+    String logName = args[0];
+
+    printLogEntries(logName);
+  }
+
+  public static final void printLogNames() {
+    // [START logging_list_logs]
```

Comment:
If you add a new region tag, you also need to ensure that the doc page that should include this snippet references this repo and this region tag. Saying as FYI - no change is needed in this code.

Reply:
Thank you. It will be the next step once PR is merged and HEAD points to the commit that has the code.